.bit/responses/2.5-Week 2 Step 5.md

```diff
@@ -7,157 +7,99 @@ step: 5
 name: Week 2 Step 5
 ---
 
-## Week 2 Step 5
+Week 2 Step 5 ⬤⬤⬤⬤⬤◯◯ | 🕐 Estimated completion: 5-10 minutes
 
-### Call the Face API P2: Using Fetch
+### ✅ Task:
+**Call the GIPHY API with the dominant emotion of a picture**
+- [ ] Create a GIPHY account
+- [ ] Generate an API key and save it as a Function App Secret
+- [ ] Create a new Azure function called `emotionHandler`

```

Comment:
Could you make it so that everything happens in one azure function? (That way students don't have to call another azure function from the first one)